pkgs/development/python-modules/reportlab/default.nix

```diff
@@ -22,9 +22,13 @@ in buildPythonPackage rec {
 
   buildInputs = [ ft pillow ];
 
+  # Remove all the test files that require access to the internet to pass.
```

Comment:
Is that also the case for the two that were already disabled?

Reply:
they were originally removed in 9558ab1ba933afda579dbfd986ea12718ac3b33f but I did some checking and they are failing the checkPhase executes `LC_ALL="en_US.UTF-8" ${python.interpreter} tests/runAll.py` but the tests upstream run from [within the tests folder](https://bitbucket.org/rptlab/reportlab/src/636c170214101a0d656bac95664dc63b0ebf60ef/appveyor.yml?at=default&fileviewer=file-view-default#appveyor.yml-90:91).

I'm going to do some testing and will rebase this and PTAL you.